hawkbit-ui/src/main/resources/VAADIN/themes/hawkbit/customstyles/popup-window.scss

```diff
@@ -169,7 +169,7 @@
 
   //confimation dialogue popup - Content margin adjustment
   .confirmbox-question-label {
-    margin: 0 8px 8px;
+    margin: 15px 8px 18px 8px;
```

Comment:
why did you have to adjust the margins?

Reply:
Because before the displayed text was manually put in a paragraph (possible through using "<p>%s</p>" with ContentMode.HTML). As this was changed (used ContentMode.TEXT instead), the margins coming from the paragraph now needed to be replaced, which is the reason for that change.